js/models/conversations.js

```diff
@@ -653,6 +663,11 @@
                   return sessionCipher.closeOpenSessionForDevice();
               }
             });
+        }).catch(function(error) {
```

Comment:
Doesn't this cause the error to *not* be propagated to callers of `getProfile`? Should we re-throw?